sample/library/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.squareup.anvil'
 
 if (rootProject.ext.generateDaggerFactoriesWithAnvil) {
   anvil {
-    generateDaggerFactories = true
+    generateDaggerFactoriesProperty = true
```

Comment:
I'm not a fan of having to use `Property` as a suffix. Is there another way?

If you would simply change the types in the extension from Boolean to Property, then it would only be a breaking change for kts, correct? I'd be in favor of that.

Reply:
It would be a binary breaking change + source breaking change for kotlin and non-source breaking change for Gradle/groovy. We could do the JVM name workaround to keep binary compatibility + only make it a source breaking change in kotlin if that works?

Reply:
Is binary compatibility really that important for Gradle plugins? Yes, somebody could depend on the plugin in their own plugin that they ship, but I think that use case is rare. 

I really dislike the "Property" suffix and would mind the source breaking change for kts less. I plan to ship the next version as `2.3`.

> We could do the JVM name workaround to keep binary compatibility + only make it a source breaking change in kotlin if that works?

Ah, you mean new properties with `@JvmName` and deprecate the new properties at the same time?

Reply:
Done and pretty proud of how it came out

Reply:
Wow, it's so beautiful.